debian/changelog

```diff
@@ -1,3 +1,9 @@
+kano-desktop (2.1-1) unstable; urgency=low
```

Comment:
Good spot, I was aware that this is wrong but I was waiting for the other PRs to merge, bump and then update this. It will become whatever we bump the updater to.